src/Modules/Processing/CustomPlayer/CustomPlayer.cpp

```diff
@@ -37,9 +38,42 @@ void CustomPlayer::exec() {
   if (!field || !frame || !robot) {
     return;
   }
+  if(!frame->has_ball()) return;
+  auto&& pos_ball = frame->ball();
+  if(field->enemyPenaltyAreaContains(pos_ball)) return;
+  if(robot->distTo(pos_ball)>150)
```

Comment:
Colocar nomes de variáveis descritivas para facilitar o entendimento:
```suggestion
  bool isRobotFarFromBall = robot->distTo(pos_ball)>150;
  if(isRobotFarFromBall)
```